### PR TITLE
Update guides_style_18f to 0.3.0, add search config

### PR DESCRIPTION
**Pull request by mbland**
cc: @afeld

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 name: Before you ship
-baseurl: /before-you-ship
```
Comment (afeld) [inline _config.yml]: Hmm, I kinda like this for consistency with 18F Pages...

**Comment by afeld** [comment]
Will this run into the same issue mentioned in https://github.com/18F/product-guide/pull/32#issuecomment-176565682?

File: _config.yml (hunk 1 above)
Comment (mbland) [inline _config.yml]: Oh, in that case, I can add it back, but if all links are prefixed with `{{ site.baseurl }}`, everything should still work.

**Comment by mbland** [comment]
Hmm, don't know why Travis is so unhappy. Looking...

File: _config.yml (hunk 1 above)
Comment (afeld) [inline _config.yml]: Confirmed locally that this change [broke the build](https://travis-ci.org/18F/before-you-ship/builds/105597827).

**Comment by mbland** [comment]
OK, Travis is happy after I restored `baseurl: /before-you-ship`, but it really shouldn't've made a difference.

**Comment by afeld** [comment]
omg so excited about having search on this site (for myself, at the very least). Thanks!

**Pull request merged**